portfolio/src/main/webapp/style.css

```diff
@@ -162,7 +162,7 @@
   opacity: 0;
   transition: .5s ease-in-out;
 }
-/* set opacaity when hovered */
```

Comment:
Since these are applied to `.photo class`, you should move this to be just after `.photo{...}`, line 155

And then leave an extra line before starting `.overlay`

Reply:
Done. I agree that this way looks better. But since `.overlay` contains the text that overlays the `.photo` div, I think the original way also makes sense.